airbyte-cdk/java/airbyte-cdk/db-destinations/src/test/kotlin/io/airbyte/cdk/integrations/destination/staging/SerialStagingConsumerFactoryTest.kt

```diff
@@ -14,7 +14,7 @@ internal class SerialStagingConsumerFactoryTest {
     @Test
     fun detectConflictingStreams() {
         val configErrorException =
-            Assertions.assertThrows(ConfigErrorException::class.java) {
```

Comment:
just curious, why switch to the explicit try-catch-fail? It seems equivalent to the assertThrows structure

Reply:
the case where the test fails doesn't show any details about the bad exception, while try/catch does.

Reply:
can you grab a screenshot of that? I tried assertThrows locally (adding a `throw IllegalArgumentException("arst")`) and it does this: (i.e. the stacktrace shows the actual exception in the `caused by`)
![image](https://github.com/user-attachments/assets/8db2f897-9821-4d3f-8024-d18ec379b97f)